cmd/soroban-cli/src/commands/contract/deploy/wasm.rs

```diff
@@ -218,25 +228,43 @@ impl NetworkRunnable for Cmd {
             salt,
             &key,
         )?;
+
         if self.fee.build_only {
+            output.check("Done!");
             return Ok(TxnResult::Txn(txn));
         }
 
+        output.info("Simulating deploy transaction…");
+
         let txn = client.simulate_and_assemble_transaction(&txn).await?;
         let txn = self.fee.apply_to_assembled_txn(txn).transaction().clone();
+
         if self.fee.sim_only {
+            output.check("Done!");
```

Comment:
```suggestion
            output.check("Simulated and prepared transaction!");
```
We use "assembled" everywhere else to mean a transaction that has been updated based the info from the simulation. So we could also call it that.